libs/partners/google-vertexai/tests/integration_tests/test_chat_models.py

```diff
@@ -174,3 +174,39 @@ def test_vertexai_single_call_fails_no_message() -> None:
         str(exc_info.value)
         == "You should provide at least one message to start the chat!"
     )
+
+
+@pytest.mark.parametrize("model_name", model_names_to_test)
+def test_chat_google_genai_system_message_error(model_name: str) -> None:
```

Comment:
nits: it looks like copy-paste artifacts, could you rename the test, please?

Reply:
+1!

Reply:
Updated test names